ts/test/test-profile-serializer.ts

```diff
@@ -41,6 +41,32 @@ import {
 
 const assert = require('assert');
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function getAndVerifyPresence(list: any[], id: number, zeroIndex = false) {
```

Comment:
zeroIndex is never passed explicitly in any invocation

Reply:
I lazily copy/pasted this from `worker.ts`.
I improved this part by sharing it in `profile-for-test.ts`